fades/pipmanager.py

```diff
@@ -83,5 +87,5 @@ def _brute_force_install_pip(self):
 
         logger.debug("Installing PIP manually in the virtualenv")
         python_exe = os.path.join(self.env_bin_path, "python")
-        helpers.logged_exec([python_exe, self.pip_installer_fname])
+        helpers.logged_exec([python_exe, self.pip_installer_fname, '-I'])
```

Comment:
I'll add a comment. Is to always ignore installed dependencies (because we always use a new virtualenv). This is necessary when you use system-site-packages